pkg/autodiscovery/autoconfig.go

```diff
@@ -219,59 +233,15 @@ func (ac *AutoConfig) HasRunOnce() bool {
 }
 
 // GetAllConfigs queries all the providers and returns all the integration
-// configurations found, resolving the ones it can
+// configurations found.
```

Comment:
> `queries all the providers`
Is this part is obsolete now?

Reply:
Resolution is done completely separate from `GetAllConfigs`. What can be resolved will be, but by a different code path now.